lib/pure/collections/tables.nim

```diff
@@ -85,7 +85,11 @@ template dataLen(t): expr = len(t.data)
 
 include tableimpl
 
-proc clear*[A, B](t: Table[A, B] | TableRef[A, B]) =
+proc clear*[A, B](t: var Table[A, B]) =
```

Comment:
No - adding "var" to the original proc signature made no difference to the error. I'm not sure why that is, but it worked if I split it into two procs instead.